src/collar/oc_anim.lsl

```diff
@@ -1,40 +1,41 @@
+// Licensed under the GPLv2.  See LICENSE for full details.
 
 
-//needs to handle anim requests from sister scripts as well
-//this script as essentially two layers
-//lower layer: coordinate animation requests that come in on link messages.  keep a list of playing anims disable AO when needed
-//upper layer: use the link message anim api to provide a pose menu
+// Needs to handle anim requests from sister scripts as well
+// This script as essentially two layers
+// Lower layer: coordinate animation requests that come in on link messages.  keep a list of playing anims disable AO when needed
+// Upper layer: use the link message anim api to provide a pose menu
 
-list g_lAnims;  //list of queued anims
-list g_lPoseList;  //list of standard poses to use in the menu
-list g_lOtherAnims; //list of animations not in g_lPoseList to forward to requesting scripts
-integer g_iNumberOfAnims;  //we store this to avoid running createanimlist() every time inventory is changed...
+list g_lAnims; // List of queued anims
+list g_lPoseList; // List of standard poses to use in the menu
+list g_lOtherAnims; // List of animations not in g_lPoseList to forward to requesting scripts
+integer g_iNumberOfAnims; // We store this to avoid running createanimlist() every time inventory is changed...
 
 //PoseMove tweak stuff
-integer g_iTweakPoseAO = 0;  //Disable/Enable AO for posed animations - set it to 1 to default PoseMove tweak to ON
-string g_sPoseMoveWalk;  //Variable to hold our current Walk animation
-string g_sPoseMoveRun;  //Variable to hold our run animation
-string g_sWalkButtonPrefix = "";  //This can be changed to prefix walks in the PoseMove menu
-list g_lPoseMoveAnimationPrefix = ["~walk_","~run_"];
+integer g_iTweakPoseAO = 0; // Disable/Enable AO for posed animations - set it to 1 to default PoseMove tweak to ON
+string g_sPoseMoveWalk; // Variable to hold our current Walk animation
+string g_sPoseMoveRun; // Variable to hold our run animation
+string g_sWalkButtonPrefix = ""; // This can be changed to prefix walks in the PoseMove menu
+list g_lPoseMoveAnimationPrefix = ["~walk_", "~run_"];
 
 string g_sCurrentPose = "";
-integer g_iLastRank = 0;  //in this integer, save the rank of the person who posed the av, according to message map.  0 means unposed
+integer g_iLastRank = 0; // In this integer, save the rank of the person who posed the av, according to message map.  0 means unposed
 integer g_iLastPostureRank = 504;
 integer g_iLastPoselockRank = 504;
-list g_lAnimButtons;  // initialized in state_entry for OpenSim compatibility (= ["Pose", "AO Menu", g_sGiveAO, "AO ON", "AO OFF"];)
+list g_lAnimButtons; // Initialized in state_entry for OpenSim compatibility (= ["Pose", "AO Menu", g_sGiveAO, "AO ON", "AO OFF"];)
 
 integer g_iAnimLock = FALSE;
-integer g_iPosture;  //posture lock on/off
+integer g_iPosture; // Posture lock on/off
 list g_lHeightAdjustments;
-integer g_iRLVA_ON;
```

Comment:
I'd prefer that acronyms be all caps, like RLVA rather than Rlva.  (Similar to Go's style rule, here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
I agree, acronyms should be in capitals.